app/questionnaire/questionnaire_manager.py

```diff
@@ -73,24 +68,24 @@ def process_incoming_answers(self, location, post_data):
 
         return is_valid
 
-    def build_state(self, location, answers):
+    def build_block_state(self, location, answers):
```

Comment:
Yeah at some point I hope we can remove it with WTForm but trying to make it maintainable in the short term